std-bits/aws/src/main/java/org/enso/aws/regions/AWSRegion.java

```diff
@@ -0,0 +1,37 @@
+package org.enso.aws.regions;
+
+import java.util.List;
+import software.amazon.awssdk.regions.Region;
+
+public class AWSRegion {
```

Comment:
Maybe
```suggestion
public final class AWSRegion {
```
to stress there shall be no way to subclass this class.